pumps/mongo.go

```diff
@@ -103,6 +107,115 @@ func (m *MongoPump) Init(config interface{}) error {
 	return nil
 }
 
+func (m *MongoPump) capCollection() {
+
+	var colName = m.dbConf.CollectionName
+	var colCapMaxSizeBytes = m.dbConf.CollectionCapMaxSizeBytes
+
+	exists, err := m.collectionExists(colName)
+	if err != nil {
+		// TODO Handle Error
+
+		return
+	}
+
+	if !exists {
+		if err := m.createCappedCollection(colName, colCapMaxSizeBytes); err != nil {
+			// TODO handle error
+
+			return
+		}
+
+		return
```

Comment:
I'd add a comment here, like `we're already capping the newly created collection`